fe/fe-core/src/main/java/org/apache/doris/common/profile/SummaryProfile.java

```diff
@@ -399,6 +415,15 @@ public void update(Map<String, String> summaryInfo) {
         updateExecutionSummaryProfile();
     }
 
+    public void queryFinished() {
+        executionSummaryProfile.addInfoString(
+                SPLITS_ASSIGNMENT_WEIGHT,
+                new GsonBuilder().disableHtmlEscaping().create().toJson(splitWeightProfileInfoMap));
+        executionSummaryProfile.addInfoString(
+                SPLITS_ASSIGNMENT_SPLIT_INFO,
+                new GsonBuilder().disableHtmlEscaping().create().toJson(splitProfileInfoMap));
```

Comment:
The number of split may be huge. It is not a good idea to print them all.
I think we can only print top 10 of them.
And also need to consider and, we'd even not save all split. Just save top 10 is enough

Reply:
Here, we don't yet know which split has the longest execution time on the BE (Backend Engine) in the end, so we can't calculate the top ones here.